index.d.ts

```diff
@@ -2,33 +2,29 @@ export { } from "./jsx";
 
 export as namespace skate;
 
-export interface HasProps<El extends Component> {
-  props: { [key: string]: skate.PropOptions<El, any>; };
-}
-
-export interface OnUpdatedCallback {
-  updatedCallback(previousProps: any): boolean | undefined | void;
-}
-
-export interface OnRenderCallback {
-  renderCallback(): any;
-}
-
-export interface OnRenderedCallback {
-  renderedCallback(): void;
-}
-
-export class Component extends HTMLElement {
-  // static readonly props: { [name: string]: PropAttr<Component, any> };
+export class Component extends HTMLElement implements ICustomElementV1, ISkateLivecycle {
```

Comment:
We should not to use `I` prefix.
My definitions does not have `ICustomElementV1` and `ISkateLivecycle` now.